uavcan/node/430.GetInfo.0.1.uavcan

```diff
@@ -0,0 +1,69 @@
+#
+# Full node info request.
+# All of the returned information must be static (unchanged) while the node is running, excepting the Status field.
+# It is highly recommended to support this service on all nodes.
+#
+
+---
+
+# The current node status (most recent heartbeat message).
+# This is the same value that is periodically published by the node.
+# The objective of its inclusion here is to ensure that the node did not restart since the last time
+# this message was received from it, because a restart may invalidate the node information.
+Heartbeat.1 most_recent_heartbeat
+
+# The UAVCAN protocol version implemented on this node, both major and minor.
+# Not to be changed while the node is running.
+Version.1 protocol_version
+
+# The version information must not be changed while the node is running.
+# The correct hardware version must be reported at all times, excepting software-only nodes, in which
+# case it should be set to zeros.
+# If the node is equipped with a UAVCAN-capable bootloader, the bootloader should report the software
+# version of the installed application, if there is any; if no application is found, zeros should be reported.
+Version.1 hardware_version
+Version.1 software_version
+
+# Reserved for future use
+void23
```

Comment:
The question is no longer applicable because I've moved stuff around, but I should answer nevertheless for the benefit of posterity: the next fields needed 1 bit for length, so 23+1 gave us 3 bytes. The preceding fields were 13 bytes long, so the resulting offset at the CRC field was 16 byte.